app/scripts/views/tour/sidebars/economy-partial.html

```diff
@@ -0,0 +1,49 @@
+<h3 class="sidebar-content-heading mode-color">
+    <span class="header-underline">
+        Types of projects:
+    </span>
+</h3>
+<div class="sidebar-content-text">
+<ul>
```

Comment:
I'm assuming this is boilerplate from the design files, but could we do something about the repeated content in these sidebar partials now?